templates/frontend/components/header.tpl

```diff
@@ -72,74 +72,16 @@
 
 				{* Primary site navigation *}
 				{if $currentContext}
-					<nav id="pkp_content_nav" class="pkp_navigation_primary_row" aria-label="{translate|escape key="common.navigation.site"}">
-						<div class="pkp_navigation_primary_wrapper">
+					<div class="pkp_navigation_primary_wrapper">
```

Comment:
It looks like the `<nav class="pkp_navigation_primary_row" aria-label="{translate|escape key="common.navigation.site"}">` tag got removed here.

Let's restore the `<nav>` wrapper to the `header.tpl` and take it out of the `navigationMenu.tpl` template. The `<nav>` wrapper needs a sensible `aria-label`, and I would only expect a technical person to handle that properly anyway. And not every menu should necessarily be wrapped in a `<nav>`.

Reply:
OK